### PR TITLE
apis: update cpu scheduling plugin args and apis

### PR DESCRIPTION
**Pull request by eahydra**
Signed-off-by: Joseph <joseph.t.lee@outlook.com>

### V. Checklist

- [x] I have written necessary docs and comments
- [x] I have added necessary unit tests and integration tests
- [x] All checks passed in `make test`

### CODE REVIEW

**Comment by jasonliu747** [comment]
Hi @eahydra, nice work. Please complete the checklist in your PR description as well.

File: apis/scheduling/config/types.go [hunk 1]
```diff
@@ -43,3 +44,62 @@ type LoadAwareSchedulingArgs struct {
...
+	// CPUBindPolicyConstrainedBurst constrains the CPU Shared Pool range of the Burstable Pod
+	CPUBindPolicyConstrainedBurst CPUBindPolicy = "ConstrainedBurst"
+)
+
+// NUMAAllocateStrategy indicates how to choose satisfied NUMA Nodes
+type NUMAAllocateStrategy string
+
+const (
+	// NUMAAllocateStrategyMostAllocated indicates that allocates from the NUMA Node with the least amount of available resource.
+	NUMAAllocateStrategyMostAllocated NUMAAllocateStrategy = "MostAllocated"
```
Comment (jasonliu747) [inline apis/scheduling/config/types.go:100]: `NUMAAllocateStrategyMostAllocated` -> `NUMAMostAllocated`, WDYT?

Comment (eahydra) [reply apis/scheduling/config/types.go:100]: nice!

Comment (eahydra) [reply apis/scheduling/config/types.go]: done

**Review by jasonliu747 — COMMENTED** [review]
/lgtm

**Pull request merged**